source/tutorials/read-joining.rst

```diff
@@ -0,0 +1,223 @@
+joined reads are (we’ll come back to this when we denoise with Deblur). When
+looking at the quality plots in this visualization, if you hover over a
+specific position you’ll see how many reads are at least that long (of the
+reads that were sampled for computing sequence quality). Make note of the
+highest sequence position where most (say, > 99%) of your reads are at least
+that long.
+
+For example, when hovering over a black box in this visualization (which is
+generated from a larger data set than the one used in this tutorial), I see
+that 10000 out of the 40126 sequences were used to estimate the quality score
+distribution at this position.
+
+When I hover over position 250, which is illustrated with a red box, I see that
+some sequences are not this long because only 9994 sequences were used for
+estimating the quality score distribution at this position. The red box and the
+red text below tell me that some sequences were not at least this long.
+
+When I hover over position 254, which is also illustrated with a red box, I see
+that many sequences are not this long because only 845 sequences were used for
+estimating the quality score distribution at this position.
+
+**Based on a comparison of these plots, I will note that most of my sequences
+are at least 250 bases long.** We plan to simplify this process in `the near
+future`_.
+
+Sequence quality control
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next we’ll apply quality control to our sequences using ``quality-filter
+q-score-joined``. This method is identical to ``quality-filter q-score``,
+except that it operated on joined reads. The parameters to this method have not
+been extensively benchmarked on joined read data, so we recommend experimenting
+with different parameter settings.
+
+.. command-block::
+
+   qiime quality-filter q-score-joined \
+     --i-demux demux-joined.qza \
+     --o-filtered-sequences demux-joined-filtered.qza \
+     --o-filter-stats demux-joined-filter-stats.qza
+
+At this stage you can choose to proceed using `Deblur`_ for additional quality
+control, or you can dereplicate sequences and optionally cluster them into OTUs
+with ``q2-vsearch``. Deblur should give much higher quality results, so we
+recommend that procedure and will illustrate that approach in the next steps of
+this tutorial.
+
+If you are instead interested in experimenting with an analysis workflow that
+is more like QIIME 1 processing (for example, to compare your Deblur or DADA2
+result with a QIIME 1-like pipeline), you should next dereplicate and cluster
+your sequences. If you try this option, we strongly encourage you to call
+``qiime quality-filter q-score-joined`` with a higher ``min-quality`` threshold
+- possibly ``--p-min-quality 20`` or ``--p-min-quality 30``. You can then
+follow the steps in the `OTU clustering tutorial`_. After clustering, you will
+likely want to filter features that are observed in only one sample using
+``qiime feature-table filter-features --p-min-samples``.
```

Comment:
setting an abundance filter is also advisable (see my ancient work linked above). At least filter out singletons!

Chimera filtering is also strongly advised with OTU picking.